lib/active_fedora/noid/synchronized_minter.rb

```diff
@@ -1,35 +1,19 @@
 require 'noid'
+
 module ActiveFedora
```

Comment:
Instead of making changes directly in this file, how would you feel about breaking your changes into a new class (e.g., `SynchronizedDatabaseBasedMinter`) and renaming this file with its original contents to e.g., `SynchronizedFileBasedMinter`?